lid.py

```diff
@@ -66,7 +70,7 @@ def main():
     output_filename = f"{clean_dataset_name}_{split}_{clean_model_name}_predictions.csv"
     output_path = predictions_dir / output_filename
 
-    df[["inputs_lid", "targets_lid"]].to_csv(
+    df[["inputs_lid", "inputs_lid_proba", "targets_lid", "targets_lid_proba"]].to_csv(
```

Comment:
@AMR-KELEG The aim here is to output both the identified language as well as its probability, as that should be available for all language identification models/systems. It can also then allow us to make a more informed decision on what's actually mislabeled and what's not.

cc @mecsabb

Reply:
I agree, better to move fast at this stage.  I'm happy to approve / merge - thoughts @AMR-KELEG?